leases/utils.py

```diff
@@ -66,6 +66,16 @@ def calculate_season_end_date(lease_end: date = None) -> date:
     return default
 
 
+def calculate_prev_season_start_date() -> date:
+    today = date.today()
+    return date(day=10, month=6, year=today.year - 1)
```

Comment:
I might put these constant into settings (e.g. `BERTH_SEASON_START_MONTH` and `BERTH_SEASON_END_MONTH`). But maybe that's not totally necessary.

Reply:
I thought that too, but I think it should need a greater refactor by then. Also, I started to think what we could really achieve with them and what would happen to the history data if the constant would be changed. Anyway, I created a new ticket of this: VEN-1507 (https://helsinkisolutionoffice.atlassian.net/browse/VEN-1507).